schemas/ticket.py

```diff
@@ -0,0 +1,33 @@
+from ma import ma
+from models.tickets import TicketModel
+from models.tenant import TenantModel
+from models.user import UserModel
+from utils.time import time_format
+from marshmallow import fields, validates, ValidationError
+
+
+class TicketSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = TicketModel
+        include_fk = True
+
+    tenant = fields.Nested("TenantSchema")
+    user = fields.Nested("UserSchema")
```

Comment:
Same here. I don't think the relationships are setup on User, however lets leave this in as we will want it.